game/scripts/npc/abilities/bristleback_bristleback_oaa.txt

```diff
@@ -26,7 +26,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "0.5"
+    "AbilityCooldown"                                     "0.4"
```

Comment:
Change cooldown in `bristleback_bristleback.txt` too so picking screen can show the correct value.